### ISSUE TITLE
What is the desired threshold for an "open" data license?

### ISSUE THREAD

**Issue by owenscott** [openboundaries/principles#6]

**Comment by bcarancibia** [comment]
One of the issues I have been reading about is the ODbL that OSM has currently in place. Maybe it would be a good thing to talk to Mikel and the larger group about, but the ODbL from OSM seems to be keeping OSM from interacting with big companies like foursquare. See this well written blogpost here: http://www.openstreetmap.org/user/lxbarth/diary/21221. This has the potential to be a real issue if the idea of this group to expand much beyond the members in the open boundaries working group.

**Comment by mikelmaron** [comment]
It's an issue for some companies, but not all, and under certain circumstances. Note Alex's post was pretty controversial, and there's a lot of emotional rhetoric. There's actually some legal research on now, to anchor the ODbL discussion more steadily.

I think it would be good for us to think through what might be the issues (or non-issues) for boundaries and OSM.

And I think it could be entirely reasonable to use OSM infrastructure, but with a different database, for a boundaries commons.
 
- Mikel Maron * +14152835207 @mikel s:mikelmaron

**Comment by bcarancibia** [comment]
I think that's totally fair, there was emotional rhetoric in it, but I think it is necessary to think about these issues when we are putting our principles together.

**Comment by tomfitzwater** [comment]
In terms of licensing, is it worth maintaining a commingled set of data? Eg, establish a bare minimum standard that data must be licensed at least comparably to CC BY-NC-SA or BY-NC-ND? Anything with ShareAlike or No derivatives likely cannot be incorporated into the final "OpenBoundaryMap" data set, but there could still be a "DMZ" of sorts where clearly licensed semi-open data can live as separate files to demonstrate the level of detail available country by country. That structure could provide a useful repository for users based on their needs, but could also give users motivation to find truly open data as well as give the project something to which it can compare contributions. E.g., if User X contributes a data set which is a 99.9% match to an existing BY-NC-ND data set in the DMZ, it may not a
be appropriate to incorporate.

**Comment by owenscott** [comment]
A good starting point for something objective might be the OKFN definition of Open:

http://opendefinition.org/od/

If you look at their list of "approved" licenses they explicitly break out attribution and share-alike requirements:

http://opendefinition.org/licenses/

So maybe a way to frame this as three separate questions are:

1) Are we willing to start from the place that a minimum license should conform to something like the Open Definition (which would include OdBL). (this would be one rating criteria or "principle")
2) Do we want to have an additional specific preference for attribution requirements, or lack thereof (this would be a second rating criteria/principle)
3) Do we want to have an additional specific preference for share-alike requirements, or lack thereof (this would be a third rating criteria/principle)

Personally my answers would be:

1) Yes. I think the OKFN's Open Definition provides a good minimum standard.
2) No. I think most data users would be comfortable with an attribution requirement.
3) Yes. Share-alike seems to be enough of a pain-point for some people to develop a specific stance on it.

Thoughts?

**Comment by mikelmaron** [comment]
@tomfitzwater I strongly believe that commercial use should be a minimum. This creates lots of problems, the line is actually pretty blurry when it comes to data.

I do like the idea of a totally comingled, who cares about licensing, data gathering exercise. Let's attract the take down notices, and see how much the licensing really matters.

**Comment by mikelmaron** [comment]
@owenscott agreed. OpenDefinition is good. Everyone is ok with attribution. Share-alike is a something to measure on data sets. I don't know if that means we take a stance or preference however.
